models/apiInvalidResponse.go

```diff
@@ -0,0 +1,18 @@
+package models
+
```

Comment:
this is all for the error handling pr. This should NOT be here. I will likely keep it here for the sake of not having to delete this and then rewrite it. But yes, this is BAD to put here for this PR.